apps/notifications/package.json

```diff
@@ -18,10 +18,11 @@
 	"homepage": "https://github.com/Automattic/wp-calypso",
 	"scripts": {
 		"clean": "npx rimraf dist",
-		"build": "yarn clean && BROWSERSLIST_ENV=wpcom calypso-build --config='./webpack.config.js'",
+		"build:notifications": "calypso-build --config='./webpack.config.js'",
 		"dev-server": "webpack serve --inline --progress --liveReload --content-base dist --watch-content-base --host notifications.localhost --port 8888",
-		"start": "yarn run clean && yarn run build && yarn run dev-server",
-		"sync": "yarn run clean && yarn run build && sh ./bin/wpcom-watch-and-sync.sh"
+		"start": "yarn run clean && yarn run build:notifications && yarn run dev-server",
+		"dev": "node ../../bin/build-calypso-app.mjs --localPath dist --remotePath /home/wpcom/public_html/widgets.wp.com/notifications",
+		"build": "NODE_ENV=production yarn dev"
```

Comment:
Why does `build` call `dev`? Isn't `build` supposed to build for production?

Reply:
Or maybe I misunderstood the semantics of those commands 🤔

Reply:
The only difference between the two modes  is `NODE_ENV`. `NODE_ENV` is the only thing which makes a difference either way under the hood. For example, if node env is development (which is the default for these scripts), then webpack will start in watch mode. If it is production, it will run a single build.